.ci/docs

```diff
@@ -1,9 +1,11 @@
 pipeline {
-    agent { label 'docs' }
+    agent {
+        label 'docs'
+    }
     options {
         timestamps()
         ansiColor('xterm')
-        timeout(time: 6, unit: 'HOURS') 
+        timeout(time: 2, unit: 'HOURS')
```

Comment:
you want to timeout after 2 hours? i don't know that any test for 2018.3 is going to finish in that time limit.

Reply:
oh this is for docs, nevermind